### PR TITLE
Move things around for 2020

### PR DESCRIPTION
**Pull request by auscompgeek**

### CODE REVIEW

File: networktables/__init__.py [hunk 1]
```diff
@@ -1,7 +1,4 @@
-# This is imported first to avoid circular dependency problems
-from .version import __version__
+from pynetworktables import *
+from pynetworktables import __version__
 
-#: Alias of NetworkTablesInstance.getDefault()
-from .networktables import NetworkTables
-
-from .instance import NetworkTablesInstance
+__all__ = ("NetworkTables", "NetworkTablesInstance")
```
Comment (virtuald) [inline networktables/__init__.py]: Shouldn't this `__all__` match the one in `pynetworktables.__init__.py`?

Comment (auscompgeek) [reply networktables/__init__.py]: I'm deliberately saying here that if someone comes along and does `from networktables import *`, they just get these two. I guess I just have this here to say "this is the API that is useful".

Comment (virtuald) [reply networktables/__init__.py]: But what about typing?

Comment (auscompgeek) [reply networktables/__init__.py]: If they want the type hints, then users can explicitly refer to them (either by explicitly listing them in the import, or accessing them via the module).

Comment (virtuald) [reply networktables/__init__.py]: Oh, right.

**Pull request merged**